### PR TITLE
Implement exercise 'reverse-string'

### PR DESCRIPTION
**Pull request by ingydotnet**

### CODE REVIEW

File: exercises/practice/reverse-string/.meta/tests.toml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+description = "a sentence with punctuation"
+
+[1f8ed2f3-56f3-459b-8f3e-6d8d654a1f6c]
+description = "a palindrome"
+
+[b9e7dec1-c6df-40bd-9fa3-cd7ded010c4c]
+description = "an even-sized word"
+
+[1bed0f8a-13b0-4bd3-9d59-3d0593326fa2]
+description = "wide characters"
```
Comment (m-dango) [inline exercises/practice/reverse-string/.meta/tests.toml:31]:
```suggestion
description = "wide characters"
include = false
```

File: exercises/practice/reverse-string/.meta/tests.toml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+[93d7e1b8-f60f-4f3c-9559-4056e10d2ead]
+description = "grapheme cluster with pre-combined form"
```
Comment (m-dango) [inline exercises/practice/reverse-string/.meta/tests.toml:35]:
```suggestion
description = "grapheme cluster with pre-combined form"
include = false
```

File: exercises/practice/reverse-string/.meta/tests.toml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+[1028b2c1-6763-4459-8540-2da47ca512d9]
+description = "grapheme clusters"
```
Comment (m-dango) [inline exercises/practice/reverse-string/.meta/tests.toml:39]:
```suggestion
description = "grapheme clusters"
include = false
```

**Review by m-dango — COMMENTED** [review]
These suggestions likely won't do much now but they'll be useful for future exercise generation.

File: exercises/practice/reverse-string/.meta/tests.toml (hunk 1 above)
Comment (ingydotnet) [reply exercises/practice/reverse-string/.meta/tests.toml:31]: Thanks for that. I was going to comment those out but this is better.

**Pull request merged**